provisioner/aws.go

```diff
@@ -756,21 +756,8 @@ func userDataConfig(stackName, stackVersion, kubeletSecret string, cluster *api.
 }
 
 // getUserData reads userdata from clc files and uploads the userdata to S3.
-func (a *awsAdapter) getUserData(basePath string, config map[string]string, bucketName string) (string, string, error) {
-	userDataMasterPath := path.Join(basePath, "master.clc.yaml")
-	userDataWorkerPath := path.Join(basePath, "worker.clc.yaml")
-
-	master, err := a.prepareUserData(userDataMasterPath, config, bucketName)
-	if err != nil {
-		return "", "", err
-	}
-
-	worker, err := a.prepareUserData(userDataWorkerPath, config, bucketName)
-	if err != nil {
-		return "", "", err
-	}
-
-	return master, worker, nil
+func (a *awsAdapter) getUserData(_ string, _ map[string]string, _ string) (string, string, error) {
+	return "", "", nil
```

Comment:
userdata in legacy masters and workers doesn't matter as their ASG is scaled to 0.